samples/jobs/jobs.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {google} = require('googleapis');
+const sampleClient = require('../sampleclient');
+
+const PROJECT_ID = process.env.GOOGLE_CLOUD_PROJECT;
+const jobService = google.jobs({
+  version: 'v3',
+  auth: sampleClient.oAuth2Client,
+});
+
+async function runSample() {
+  const res = await jobService.projects.companies.create({
+    parent: `project/${PROJECT_ID}`,
+    requestBody: {
+      company: {
+        displayName: 'ABC co.',
+        externalId: '12345',
+      },
+    },
+  });
+  console.log(res.data);
+  return res.data;
+}
+
+if (module === require.main) {
+  const scopes = [
+    'https://www.googleapis.com/auth/jobs',
+    'https://www.googleapis.com/auth/cloud-platform',
+  ];
+  sampleClient
+    .authenticate(scopes)
+    .then(runSample)
+    .catch(console.error);
+}
+
+module.exports = {
```

Comment:
It would be nice to have a test for this under `test/samples` :)

Reply:
I'll get it to you ASAP! Thanks!